app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<h1 class="text-center p-4 mb-3">User List</h1>
+
+<div class="row">
+    <% @users.each do |user| %>
+        <div class="col-lg-12 mb-3">
+            <div class="media">
+                <div class="mr-3 border">
+                    <%= link_to gravatar_for(user) %>  
+                </div> 
+                <div class="media-body">
+                    <h5><%= link_to user.name,user_path(user) %></h5>
+                </div>
+                <% unless current_user == @user %>
+                    <% if user.admin? %>
+                        <%= link_to "Remove as Admin", admin_user_path(user), method: :patch, class: 'btn btn-outline-danger',
+                            data: { confirm: "Are you sure?" } %>
+                    <% else %>
+                        <%= link_to "Set as Admin", admin_user_path(user), method: :put, class: 'btn btn-outline-primary',
+                            data: { confirm: "Are you sure?" } %>
+                    <% end %>
+                <% end %>
```

Comment:
We dont usually put if else statements in our view. Try to make a helper method for this functionality.